docs/kratos/self-service/flows/account-recovery-password-reset.mdx

```diff
@@ -160,12 +160,23 @@ In the Ory Console go to <ConsoleLink route="project.mfa" />
 ### Account recovery address
 
 To start account recovery, Ory Identities must know which address to send the recovery message to. Usually this is the email
-address the user provides when registering their account. Other fields inside the `traits` section are supported as well.
+address the user provides when registering their account. Other fields inside the `traits` section are supported as well such a
+phone number to receive the code via a SMS.
+
+If there are multiple recovery addresses registered for the user and if the feature flag `choose_recovery_address` is
+[enabled](../../../identities/get-started/account-recovery.mdx#enable-the-feature-flag-choose_recovery_address-unlocks-sending-a-recovery-code-via-sms),
+entering any of them will present them a list of (masked) addresses to choose from (up to 10). Once they have picked one address,
+and entered the address fully (to demonstrate that they indeed know the address and also to avoid information exfiltration), this
+address is used. The address can be an email address, in which case an email is sent, or a phone number, in which case a SMS is
+sent.
```

Comment:
```suggestion
```suggestion
If a user has multiple recovery addresses and the `choose_recovery_address` feature flag is [enabled](../../../identities/get-started/account-recovery.mdx#enable-the-feature-flag-choose_recovery_address-unlocks-sending-a-recovery-code-via-sms), the process is as follows:

1. The user enters any of their registered addresses to begin the recovery flow.
1. A masked list of their recovery addresses is displayed. Up to 10 addresses are shown.
1. After selecting an address, the user must enter it in full. This step verifies ownership and helps prevent information leaks.
1. The recovery message is sent to the confirmed address via email or SMS.
```
```